libs/librepcb/common/graphics/graphicslayer.cpp

```diff
@@ -232,54 +232,82 @@ void GraphicsLayer::getDefaultValues(const QString& name, QString& nameTr,
     h.insert(sSymbolPinNumbers,         {tr("Pin Numbers"),           QColor(64, 64, 64, 255),    Qt::gray,                   true});
     // board asymmetric
-    h.insert(sBoardSheetFrames,         {tr("Sheet Frames"),          Qt::lightGray,              Qt::white,                  true});
-    h.insert(sBoardOutlines,            {tr("Board Outlines"),        QColor(255, 255, 255, 180), QColor(255, 255, 255, 220), true});
-    h.insert(sBoardMillingPth,          {tr("Milling (PTH)"),         QColor(0, 255, 255, 150),   QColor(0, 255, 255, 220),   true});
-    h.insert(sBoardDrillsNpth,          {tr("Drills (NPTH)"),         QColor(255, 255, 255, 200), QColor(255, 255, 255, 255), true});
-    h.insert(sBoardPadsTht,             {tr("Pads"),                  QColor(0, 255, 0, 150),     QColor(0, 255, 0, 220),     true});
-    h.insert(sBoardViasTht,             {tr("Vias"),                  QColor(0, 255, 0, 150),     QColor(0, 255, 0, 220),     true});
+    h.insert(sBoardSheetFrames,         {tr("Sheet Frames"),          QColor("#96E0E0E0"),        QColor("#FFFFFFFF"),        true});
+    h.insert(sBoardOutlines,            {tr("Board Outlines"),        QColor("#C8FFFFFF"),        QColor("#FFFFFFFF"),        true});
+    h.insert(sBoardMillingPth,          {tr("Milling (PTH)"),         QColor("#C8FFFFFF"),        QColor("#FFFFFFFF"),        true});
+    h.insert(sBoardDrillsNpth,          {tr("Drills (NPTH)"),         QColor("#C8FFFFFF"),        QColor("#FFFFFFFF"),        true});
+    h.insert(sBoardPadsTht,             {tr("Pads"),                  QColor("#966DB515"),        QColor("#B44EFC14"),        true});
+    h.insert(sBoardViasTht,             {tr("Vias"),                  QColor("#966DB515"),        QColor("#B44EFC14"),        true});
     h.insert(sBoardAirWires,            {tr("Air Wires"),             Qt::yellow,                 Qt::yellow,                 true});
-    h.insert(sBoardMeasures,            {tr("Measures"),              Qt::gray,                   Qt::lightGray,              true});
-    h.insert(sBoardAlignment,           {tr("Alignment"),             Qt::darkCyan,               Qt::cyan,                   true});
-    h.insert(sBoardDocumentation,       {tr("Documentation"),         Qt::lightGray,              Qt::white,                  true});
-    h.insert(sBoardComments,            {tr("Comments"),              QColor(255, 165, 0, 180),   QColor(255, 165, 0, 255),   true});
-    h.insert(sBoardGuide,               {tr("Guide"),                 Qt::darkYellow,             Qt::yellow,                 true});
+    h.insert(sBoardMeasures,            {tr("Measures"),              QColor("#FF808000"),        QColor("#FFA3B200"),        true});
+    h.insert(sBoardAlignment,           {tr("Alignment"),             QColor("#B4E59500"),        QColor("#DCFFBF00"),        true});
+    h.insert(sBoardDocumentation,       {tr("Documentation"),         QColor("#B4E59500"),        QColor("#DCFFBF00"),        true});
```

Comment:
IMHO this layer looks better when using the same color as for top and bottom documentation:

```suggestion
    h.insert(sBoardDocumentation,       {tr("Documentation"),         QColor("#96E0E0E0"),        QColor("#DCE0E0E0"),        true});
```

Reply:
Do it! (I can't commit the suggestion for some reason...)